docs/introduction-to-smart-contracts.rst

```diff
@@ -304,7 +304,7 @@ These blocks form a linear sequence in time, and that is where the word "blockch
 Blocks are added to the chain at regular intervals, although these intervals may be subject to change in the future.
 For the most up-to-date information, it is recommended to monitor the network, for example, on `Etherscan <https://etherscan.io/chart/blocktime>`_.
 
-As part of the "order selection mechanism" (which is called "mining") it may happen that
+As part of the "order selection mechanism" (which is called "attestation") it may happen that
```

Comment:
Perhaps we could also make this a link to https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/attestations/